pkg/resourcetotelemetry/resource_to_telemetry.go

```diff
@@ -32,15 +35,22 @@ import (
 //	}
 type Settings struct {
 	// Enabled indicates whether to convert resource attributes to telemetry attributes. Default is `false`.
-	Enabled bool `mapstructure:"enabled"`
+	Enabled bool            `mapstructure:"enabled"`
+	Exclude MatchAttributes `mapstructure:"exclude"`
```

Comment:
Should this be `filter`? This does not match the configuration example in the README

Reply:
Readme fixed - I changed filter to exclude to be similar to other excludes:

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/filterprocessor/config.go#L53
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/coreinternal/processor/filtermetric/config.go#L40

WDYT?

Reply:
Make sense to me, I want to take a closer look next week but this schema looks great and is consistent with other components.